### ISSUE TITLE
Poll + delete in a single call?

### ISSUE THREAD

**Issue by jeromegn** [wilsonzlin/queued#1]
This is a very interesting project.

Would it be possible to poll + delete in a single call (like adding a delete option to the poll request) or is that a constraint of the architecture?

**Comment by wilsonzlin** [comment]
Yes, that's possible. It's currently not implemented but should be reasonably easy to add. It does mean that if the poller loses the message, it's gone forever.
